Messenger/AppDelegate.mm

```diff
@@ -173,6 +168,27 @@ - (void)applicationDidBecomeActive:(NSNotification *)notification {
 }
 
 
+#pragma mark - Javascript events
+
+- (id)evaluateWebScript:(NSString *)script {
```

Comment:
Let's not do this. It saves very little typing and adds one frame to debugging and tracing.